daphne_worker/src/dap.rs

```diff
@@ -157,20 +157,28 @@ impl<D> DapAggregator<BearerToken> for DaphneWorkerConfig<D> {
         // checking the AggregateStore and seeing whether it requests for aggregate
         // shares that have already been marked collected.
         let namespace = self.durable_object("DAP_AGGREGATE_STORE")?;
+        let mut stubs: Vec<Stub> = Vec::new();
         for window in (batch_interval.start..batch_interval.end())
             .step_by(task_config.min_batch_duration.try_into().unwrap())
         {
-            let stub = namespace
-                .id_from_name(&durable_agg_store_name(
-                    &task_config.version,
-                    &task_id.to_hex(),
-                    window,
-                ))?
-                .get_stub()?;
+            stubs.push(
```

Comment:
This approach might work, but it has the disadvantage of adding a lot of code complexity. If you we need to keep the stub in scope, you could have the async function you call create it. In fact, this is what we already do with `DaphneWorkerConfig::durable_post()`. (I think deleting this method is a regression.)

Reply:
I've reverted the changes and brought back `DaphneWorkerConfig::durable_post()`.